test_schemas/en/test_summary_with_submission_text.json

```diff
@@ -21,6 +21,14 @@
             "type": "string"
         }
     ],
+    "questionnaire_flow": {
```

Comment:
This file should be renamed to make it clear it's testing submit e.g. `test_submit_with_summary_custom_submission_text.json`